### PR TITLE
import: remove last dependency on symlink

### LINKED ISSUE #3479: How do we lxd import? Instructions are extremely unclear
I use Ubuntu 16.04, and all i got for importing instructions was the help screen when I type just: lxd import

To test this feature, I had already done a backup of a container from the containers directory, with zfs send | zfs receive.

Then I targz-ed the directory I sent, as root user in order to preserve file ownerships. I took it to a different machine, with a fresh install of the newest lxd (2.15) just like the source machine's lxd.

I didn't know where to untar it, because the instructions were so unclear. So I created a zfs filesystem called test-container under default/containers in the destination machine zfs pool. I untarred (as root again), and did a "mv" of the contents of the untarred directory inside the default/containers/test-container (that was the original name of the container, as well).

Then I did: lxd import test-container

and got:

root@test-vm:/# lxd import test-container
error: The container "test-container" does not seem to exist on any storage pool.

What should I do? The instructions ought to be a lot clearer than this...

### PR DESCRIPTION
**Pull request by brauner**
Closes #3479.

Signed-off-by: Christian Brauner <christian.brauner@ubuntu.com>

### CODE REVIEW

File: lxd/api_internal.go [hunk 1]
```diff
@@ -183,7 +183,8 @@ func internalImport(d *Daemon, r *http.Request) Response {
 	}
 
 	// Read in the backup.yaml file.
-	backup, err := slurpBackupFile(shared.VarPath("containers", req.Name, "backup.yaml"))
+	backupYamlPath := shared.VarPath("storage-pools", "containers", req.Name, "backup.yaml")
```
Comment (stgraber) [inline lxd/api_internal.go]: That path is missing the pool name.

**Comment by stgraber** [comment]
Can we get a test for this? should just be a matter of removing the symlink as well as the container entry right?

File: lxd/api_internal.go (hunk 1 above)
Comment (brauner) [inline lxd/api_internal.go]: Ugh, sorry. Too late here.

**Comment by stgraber** [comment]
Looks good. Can you maybe split this into two separate commits though?
The commit message doesn't really cover everything that's happening in there :)

**Pull request merged**